roles/gateway/README.rst

```diff
@@ -23,11 +23,15 @@ A list of packages to install that is specific to the role.  The list is defined
 
     packages: []
 
+A unique name to give to your OpenVPN service.  This name is used to organize configuration files and start/stop the service.  Defined in the secrets repo::
+
+    openvpn_server_name: []
```

Comment:
If this isn't given a default, is there something which will check for it being set and error if not?